### PR TITLE
Update changelog for 0.27.0.

### PR DESCRIPTION
**Pull request by sean-mcmanus**
Removed all the "insider-only" issues.

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -72,8 +28,20 @@
...
 * Fix issue with providing custom configurations for files specified using URIs schemes we do not recognize. [#4889](https://github.com/microsoft/vscode-cpptools/issues/4889)
+* Fix Outline view not updating fast enough after switching branches. [#4894](https://github.com/microsoft/vscode-cpptools/issues/4894)
 * Fix failure to detect CL.exe if VS Installer files are stored on a drive other than the system drive. [#4929](https://github.com/microsoft/vscode-cpptools/issues/4929)
+* Fix extension randomly getting stuck while communicating with the IntelliSense process on Mac. [#4989](https://github.com/microsoft/vscode-cpptools/issues/4989)
+* Fix completion results appearing after numeric literals. [#5019](https://github.com/microsoft/vscode-cpptools/issues/5019)
 * Fix issue with cancellation of a `Rename` operation causing subsequent `Find All References` and `Rename` operations to fail.  [#5022](https://github.com/microsoft/vscode-cpptools/issues/5022)
+* Fix some settings not being editable in the UI. [PR #5126](https://github.com/microsoft/vscode-cpptools/pull/5126)
```
Comment (michelleangela) [inline Extension/CHANGELOG.md:36]: Is this from 0.26.3 or due to changes in insiders?

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -72,8 +28,20 @@
...
+* Fix `cpp_properties.json` error squiggles not appearing. [#5131](https://github.com/microsoft/vscode-cpptools/issues/5131)
+* Fix `search.exclude` not applying if there are > 1 symbols matching in the excluded file. [#5152](https://github.com/microsoft/vscode-cpptools/issues/5152)
+* Fix tag parsing not working on Windows 7 without SP1. [#5155](https://github.com/microsoft/vscode-cpptools/issues/5155)
+* Fix `updateChannel` being settable per-workspace. [PR #5185](https://github.com/microsoft/vscode-cpptools/pull/5185)
```
Comment (michelleangela) [inline Extension/CHANGELOG.md:40]: Is this also due to insiders or part of multiroot changes?

**Review by michelleangela — APPROVED** [review]

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md:36]: It repros in 0.26.3.

File: Extension/CHANGELOG.md (hunk 2 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md:40]: Repros with 0.26.3 -- you need to have a workspace and not just a workspace folder.

**Pull request merged**